code/game/objects/structures/detectiveboard.dm

```diff
@@ -0,0 +1,254 @@
+#define MAX_ICON_NOTICES 5
+
+/obj/structure/detectiveboard
+	name = "detective notice board"
+	desc = "A board for linking evidence to crimes."
+	icon = 'icons/obj/wallmounts.dmi'
+	icon_state = "noticeboard"
+	density = FALSE
+	anchored = TRUE
+	max_integrity = 150
+	req_access = ACCESS_DETECTIVE
+
+	var/attaching_evidence = FALSE
+	var/list/case_colors = list("red", "orange", "yellow", "green", "blue", "violet")
+	var/list/datum/case/cases = list()
+	var/current_case = 1
+	var/data_test
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/detectiveboard, 32)
+
+/obj/structure/detectiveboard/Initialize(mapload)
+	. = ..()
+
+	if(!mapload)
+		return
+	for(var/obj/item/I in loc)
+		if(istype(I, /obj/item/paper))
+			I.forceMove(src)
+			cases[current_case].notices++
+
+/// Attaching evidences: photo and papers
+
+/obj/structure/detectiveboard/attackby(obj/item/O, mob/user, params)
+	if(!cases.len)
+		to_chat(user, "There are no cases!")
+		return
+	if(istype(O, /obj/item/paper) || istype(O, /obj/item/photo))
+		if(attaching_evidence)
+			to_chat(user, "You already attaching evidence!")
+			return
+		if(!allowed(user))
+			to_chat(user, span_warning("You are not authorized to add notices!"))
+			return
```

Comment:
Is this sensical/reasonable restriction to have?

Reply:
Maybe no, because everyone can take or add evidences